### PR TITLE
Serverrefactor

### PR DESCRIPTION
**Pull request by cemac-ccs**
Refactor of serverpi code to use SensorMod repo, along with prep for sharepoint upload.
Code for OLED also included

### CODE REVIEW

File: rc.local [hunk 1]
```diff
@@ -28,8 +28,6 @@ while [[ $count -ne 0 ]] ; do
 done
 
 if [[ $rc -eq 0 ]] ; then                  # Make final determination.
-    sudo systemctl stop ntp;
```
Comment (wolfiex) [inline rc.local:31]: Why were these removed?

Are they added in later on?

**Review by wolfiex — COMMENTED** [review]
See comment

File: rc.local (hunk 1 above)
Comment (wolfiex) [reply rc.local:31]: @cemac-ccs

File: rc.local [hunk 2]
```diff
@@ -28,8 +28,6 @@ while [[ $count -ne 0 ]] ; do
...
-    sudo ntpd -q -g >> ${LOC}/time.log
```
Comment (wolfiex) [inline rc.local:32]: NB this line syncs the clock from the internet

**Review by wolfiex — COMMENTED** [review]
@cemac-ccs see comments, if not an issue approve.

File: rc.local (hunk 1 above)
Comment (cemac-ccs) [reply rc.local:31]: Copying error. I'll add them back in

File: rc.local (hunk 2 above)
Comment (cemac-ccs) [reply rc.local:32]: Added back in

**Review by cemac-ccs — COMMENTED** [review]
Approved with changes suggested by @wolfiex

**Pull request merged**